src/suites/positive/validation/response-validator.ts

```diff
@@ -30,6 +31,23 @@ class ResponseValidator extends PositiveValidator {
         return;
       }
 
+      // if the Accept header is populated in the request
+      const acceptHeader = this.operation.parameters?.filter(
+        (parameter) => parameter.in === 'header' && parameter.name === 'Accept',
+      );
```

Comment:
Have we added a way to send in the Accept header we want? I wonder if we should just be changing the swagger client request to have the Accept header we want.  Will this code even get executed if there is no Accept Header?

Reply:
The Accept header is optional, and if it's needed I would expect it to be included in the examples.  When it's not populated the response type is not checked against it.

Reply:
Loast is the client here, and we can specify the Accept Header since that is all we can handle is "application/json" at this point in time.  The OAS specifies the different content-types that a request can respond with under the "responses/responseCode/content". It will not be expected to be an Example.
For now, for this ticket, I think we can always send in the "application/json" as the Accept Header and then we will know when that is not acceptable by the API as the result of your code changes, and can do more enhancing at that point in time.  

Future point in time, we should walk through all the content-types of a 200 response and check them all.